src/components/alert/alert-component.ts

```diff
@@ -303,6 +338,14 @@ export class AlertCmp {
   }
 
   getValues(): any {
+    if (this.instance) {
+
+      if (typeof this.instance.getValues === 'function')
```

Comment:
I am not sure we can do this in the framework. This is an important API change, basically we are adding a new `getValues()` method in components, that only works in Alert? and it is not documented

why does it work in Alert? but it does not in Nav.pop()? or it does not work in Modal?

We need to find a better solution, or writing a full API proposal for `getValues(). otherwise it is unlikely this will be merged soon